package.json

```diff
@@ -40,6 +40,7 @@
     "babel-preset-react": "~6.11.1",
     "browser-sync": "~2.15.0",
     "browser-sync-webpack-plugin": "~1.1.2",
+    "bs-fullscreen-message": "~1.1.0",
```

Comment:
hmm interesting, maybe is not necessary, I think that actually I'm using it: https://github.com/Kikobeats/react-boilerplatinum/blob/master/webpack.dev.config.js#L33 Can you confirm me ?